docs/root/intro/version_history.rst

```diff
@@ -72,6 +72,7 @@ Version history
   routing to certain hosts only when there are insufficient healthy hosts available.
 * upstream: add cluster factory to allow creating and registering :ref:`custom cluster type<arch_overview_service_discovery_types_custom>`.
 * upstream: added a :ref:`circuit breaker <arch_overview_circuit_break_cluster_maximum_connection_pools>` to limit the number of concurrent connection pools in use.
+* upstream: removed cluster.hosts.
```

Comment:
`removed deprecated clusters.hosts field in favor of :ref: link to the new field`.